src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -1952,6 +1952,26 @@ public boolean isAppInForeground() {
         return false;
     }
 
+    /**
+     * Based on the application's event lifecycle this method will determine whether the app
+     * is paused or not.
+     *
+     * If your build version is below 14 this method will always return false.
+     *
+     * @return True if the app is in paused state.
+     */
+    public boolean isAppPaused() {
+        if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.ICE_CREAM_SANDWICH) {
```

Comment:
we can safely remove this check since the creator of the lifecycle checks for the API version already and `mMixpanelActivityLifecycleCallbacks` will be null otherwise. we don't need to check everytime we access this var.